src/node_external_reference.h

```diff
@@ -146,6 +38,13 @@ class ExternalReferenceRegistry {
   ALLOWED_EXTERNAL_REFERENCE_TYPES(V)
 #undef V
 
+  // Registers both the underlying function pointer
+  // and the corresponding CFunctionInfo.
+  void RegisterCFunction(const v8::CFunction& c_func) {
```

Comment:
This can be an overload of `Register` to reduce changes in bindings:

```suggestion
  void Register(const v8::CFunction& c_func) {
```